.github/workflows/e2e-test-ci.yml

```diff
@@ -0,0 +1,42 @@
+name: APISXI-Seed E2E Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: setup go
+      uses: actions/setup-go@v2.1.5
+      with:
+        go-version: "1.17"
+
+    - name: startup apisix and nacos
+      run: |
+        docker-compose -f ci/docker-compose.yml up -d
+        sleep 5
+
+    - name: build apisix-seed
+      run: |
+        go build
+
+    - name: startup apisix-seed
+      run: |
+        nohup ./apisix-seed &
```

Comment:
Would be better to run apisix-seed as a docker too

Reply:
So we can restart it just like other component.

Reply:
> Would be better to run apisix-seed as a docker too

I think we should use the current version of the code for e2e testing. How to use docker to ensure that the code is updated in time?

Reply:
We can build it, like
https://github.com/apache/apisix/blob/6cd7b8d024d2b3da9db0a61dbff3b462059fa233/.github/workflows/chaos.yml#L42
https://github.com/apache/apisix-docker/blob/407610575b68b8764a3af787c9455e8bd7c1771f/example/docker-compose-alpine.yml#L22